images/virtualization-artifact/pkg/migration/qemu_max_length_36.go

```diff
@@ -52,16 +53,21 @@ func (r *qemuMaxLength36) Name() string {
 }
 
 func (r *qemuMaxLength36) Migrate(ctx context.Context) error {
```

Comment:
It should have a description that answer a question "why this migration is needed".
Which fields we mutate and why old values are invalid.
Also, explain the lifespan of this migration, e.g. add TODO to remember when to delete this migration or describe why it should be kept forever.

Reply:
There is a description in README